hedera-node/src/test/java/com/hedera/services/ledger/TransactionalLedgerTest.java

```diff
@@ -56,32 +53,35 @@
 import static org.mockito.BDDMockito.verifyNoMoreInteractions;
 import static org.mockito.BDDMockito.willThrow;
 
-public class TransactionalLedgerTest {
+class TransactionalLedgerTest {
 	Supplier<TestAccount> newAccountFactory;
 	BackingStore<Long, TestAccount> backingAccounts;
 	ChangeSummaryManager<TestAccount, TestAccountProperty> changeManager = new ChangeSummaryManager<>();
 	TransactionalLedger<Long, TestAccountProperty, TestAccount> subject;
 
-	TokenID tid = IdUtils.tokenWith(666L);
 	Object[] things = { "a", "b", "c", "d" };
 	MerkleToken token;
 	TestAccount account1 = new TestAccount(1L, things[1], false, 667L);
-	TestAccount account1TokenCopy = new TestAccount(1L, things[1], false, 667L);
 
 	@BeforeEach
 	private void setup() {
 		token = mock(MerkleToken.class);
 
 		backingAccounts = mock(BackingStore.class);
 		given(backingAccounts.getRef(1L)).willReturn(account1);
+		given(backingAccounts.getUnsafeRef(1L)).willReturn(account1);
 		given(backingAccounts.contains(1L)).willReturn(true);
 		newAccountFactory = () -> new TestAccount();
 
-		subject = new TransactionalLedger<>(TestAccountProperty.class, newAccountFactory, backingAccounts, changeManager);
+		subject = new TransactionalLedger<>(
+				TestAccountProperty.class,
+				newAccountFactory,
+				backingAccounts,
+				changeManager);
 	}
 
 	@Test
-	public void rollbackFlushesMutableRefs() {
+	void rollbackClearsChanges() {
```

Comment:
We are used to the test methods to be public. Is there any reason to change that?

Reply:
@ljianghedera From my observation, sonar cloud has a code smell info that says test methods need not be public

```
JUnit5 is more tolerant regarding the visibilities of Test classes than JUnit4, which required everything to be public.

In this context, JUnit5 test classes can have any visibility but private, however, it is recommended to use the default package visibility, which improves readability of code.
```

Reply:
Thanks for the explanation. I guess it's junit5's new feature. Just wondering will this bring some meaningful benefits, like better security, etc.